statsmodels/regression/recursive_ls.py

```diff
@@ -370,10 +398,21 @@ def cusum_squares(self):
            Series B (Methodological) 37 (2): 149-92.
 
         """
-        numer = np.cumsum(self.resid_recursive[self.loglikelihood_burn:]**2)
+        d = max(self.nobs_diffuse, self.loglikelihood_burn)
+        numer = np.cumsum(self.resid_recursive[d:]**2)
         denom = numer[-1]
         return numer / denom
 
+    @cache_readonly
+    def llf_obs(self):
```

Comment:
maybe better `llf_recursive`, IIUC
those are sequential loglike_obs values, and not the loglike evaluated at the final params, is it?

Reply:
Ah, this is a good idea, thank you.

Reply:
If I do this, I guess I should leave `llf` as the Kalman filter output rather than the one computed from the recursive residuals, since it might be confusing to have `np.sum(res.llf_obs) != res.llf`?